### DISCUSSION TITLE
papermc support

### DISCUSSION THREAD

**Discussion by sr972** [RakambdaOrg/FallingTree#516] [Ideas]
Hi Rakambda,
hi all,

is there a chance, that FallingTree may get a papermc support in future versions?
I would really love to see this mod on my papermc servers, as it already was great on my fabrics.

**Comment by Rakambda** [comment]
Papermc is a server and uses plugins, not really mods.
As it works in a different way there won't be support for it. Someone will have to make a version targetter for bukkit/spigot/paper.
